integration-tests/test/test_bonding_validators.py

```diff
@@ -39,61 +35,82 @@ def test_heterogenous_validators(command_line_options: CommandLineOptions, rando
     }
 
     bonded_validator_map = {
-        BOOTSTRAP_KEY: 200,
-        BONDED_VALIDATOR_KEY: 300
+        BOOTSTRAP_KEY: 10000000,
+        BONDED_VALIDATOR_KEY: 10000000
     }
 
+    epoch_length = 4
+    quarantine_length = 2
```

Comment:
Not sure it's a big deal, but quarantine length should be longer than epoch length, typically it's 5x longer.